packages/sw-routing/src/lib/express-route.js

```diff
@@ -64,14 +64,14 @@ class ExpressRoute extends Route {
    * Constructor for ExpressRoute.
    *
    * @param {Object} input
-   * @param {string} input.path The path to use for routing.
-   *        If the path contains [named parameters](https://github.com/pillarjs/path-to-regexp#named-parameters),
-   *        then an Object mapping parameter names to the corresponding value
-   *        will be passed to the handler via `params`.
-   * @param {Object} input.handler - An Object with a `handle` method that
-   *        will be used to respond to matching requests.
+   * @param {String} input.path The path to use for routing.
+   * If the path contains [named parameters](https://github.com/pillarjs/path-to-regexp#named-parameters),
+   * then an Object mapping parameter names to the corresponding value
```

Comment:
Maybe 'an Object that maps parameter names...'

The other way, I read 'names' as the verb. Clearly mapping/maps is the verb.